resources/js/components/Header.vue

```diff
@@ -1,51 +1,32 @@
 <template>
     <div class="border-black border-t-2 flex items-center">
         <router-link to="/">
-            <div class="bg-black p-2">
-                <img v-if="code" class="h-9 w-9" :src="`/img/${code}.svg`" :alt="code" />
-                <svg v-else class="h-9 w-9" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <div class="p-2.5 border-r-2" :class="[code ? 'border-r-transparent' : 'bg-green border-r-black']">
+                <svg v-if="code" class="h-4 w-auto ml-2" viewBox="0 0 10 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
My proposal for dealing with svg's is in my PR:
https://github.com/SlovakNationalGallery/bookmarks.sng.sk/pull/37/files#diff-32a27d5d23c7729f91299c44896683d5c5293286baa0fbce8a9f06850b287c58R8

I'd set dimensions explicitly in pixel units since there are stroked paths and no scaling is desired.
We also don't `viewBox` attribute or `xmlns` (https://stackoverflow.com/questions/18467982/are-svg-parameters-such-as-xmlns-and-version-needed)
```suggestion
                <svg v-if="code" class="h-[18px] w-[10px] ml-2" fill="none">
```

For other svg attributes like `fill`, `stroke`, `stroke-width`, `stroke-linecap`, `stroke-linejoin` I'd prefer to use tailwind classes.

Reply:
yes! I really liked your approach with svg-s. Applied.